src/test/utils.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render as rtlRender } from '@testing-library/react';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducer from '../reducers';
+
+function render(
```

Comment:
Are we still planning to rename this as `renderWithStore` or something similar?

Reply:
@gildub if you rename this function I'll go ahead and merge this, assuming you still plan to use it. Otherwise let's close it